package.json

```diff
@@ -320,6 +320,11 @@
           "type": "string",
           "default": "",
           "description": "The Go build tags to use for all commands that support a `-tags '...'` argument"
+        },
+        "go.blockImportsOnly": {
```

Comment:
I actually think we can just make this the default.  No need for a new setting here.